test/test_files/vortex_patch_godunov/vortex_patch.i

```diff
@@ -0,0 +1,65 @@
+#¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨#
```

Comment:
you'll have to name this file vortex_patch_godunov.i to match the folder

Reply:
unused parmparse variables:
  [TOP]::io.line_plot_int(nvals = 1)  :: [1]
  [TOP]::amr.plt_tracer(nvals = 1)  :: [1]
  [TOP]::amrex.fpe_trap_invalid(nvals = 1)  :: [0]
  [TOP]::amr.plot_file(nvals = 1)  :: [plt]